prefetch.bs

```diff
@@ -713,6 +714,18 @@ This section contains patches to [[NAVIGATION-TIMING]].
     Add an additional parameter to [=create the navigation timing entry=], which is a [=string=] <var ignore>deliveryType</var>, and pass it as an additional argument to [=setup the resource timing entry=].
 </div>
 
+<h2 id="interaction-with-other-specs">Interaction with other specifications and concepts</h2>
+
+<div algorithm="clear prefetch cache for origin patch">
+  Modify [$clear cache for origin$] to add the following step at the end.
+
+  1. [=list/For each=] [=navigable/top-level traversable=] |traversable| in the user agent's [=top-level traversable set=]:
```

Comment:
```suggestion
  1. [=list/For each=] [=navigable/top-level traversable=] |traversable| of the user agent's [=top-level traversable set=]:
```

(per https://infra.spec.whatwg.org/#list-iterate)

Reply:
Thanks, updated the usage in prerendering as well to fit the spec for list-iterate.